### ISSUE TITLE
Mobile (iOS): selection of Dropbox as sync target is not saved

### ISSUE THREAD

**Issue by ItsMeOnceMore** [laurent22/joplin#6381]
Hi,

Joplin was working fine on my iPhone until some time ago, when it stopped syncing (I mainly use Joplin on the laptop, so I don't know exactly when it stopped), though it kept giving a successful sync message.
So I decided to reinstall the app.
Then I open the freshly re-installed app, set the encryption and select Dropbox as sync target. I go back to the main menu, click on Synchronize and get thrown back to the settings, where I see that the sync target is back to None.
I repeated the procedure a few times (but not the encryption setting, which is properly saved); nothing changed. Also reinstalled the app again; nothing changed.

Joplin version 12.7.1
iPhone 6s
iOS 15.4

## Steps to reproduce
1. Install and open  Joplin
2. In the Configuration page, select Dropbox as Synchronisation target
3. In the main menu, tap on Synchronise 
4. See the Configuration page opening at the Synchronisation section, with (None) set as Synchronisation target

## Describe what you expected to happen
As for https://github.com/laurent22/joplin#dropbox-synchronisation, I expected to be guided further in setting Dropbox as sync backend.

## Logfile
The log file shows no errors

**Comment by Daeraxa** [comment]
Did you actually press the save icon to save the setting after you changed it?

**Comment by ItsMeOnceMore** [comment]
> Did you actually press the save icon to save the setting after you changed it?

I don't remember clearly anymore, but I believe that, after selecting Dropbox, the save icon didn't show up; it stayed the greyed-out checkmark. I could only tap the back arrow icon. 

In the meantime I switched to Nextcloud, using the generic WebDAV connector, and at that point I had no problem to save the new configuration. Now I tried again to change the sync to Dropbox and I can actually save the choice...
(And then I switched again to Nextcloud and now I have syncing problems because of locked files... Hope I will straight it out...).

Ok, since nobody else seems to have this problem and I don't feel like completely ruling out a moment of embarrassing blindness, I close the issue.

Sorry for the waste of time and thank you for the patience.
